tests/benchmarks/utils.py

```diff
@@ -4,15 +4,11 @@
 """Utilities for benchmark tests."""
 
 import os
-
-
-def setup_simulated_ddp_distributed_env():
-    """Function to setup the simulated DDP distributed envionment variables."""
-    os.environ['WORLD_SIZE'] = '1'
-    os.environ['RANK'] = '0'
-    os.environ['LOCAL_RANK'] = '0'
-    os.environ['MASTER_ADDR'] = 'localhost'
-    os.environ['MASTER_PORT'] = '12345'
+import socket
+from contextlib import closing
+import multiprocessing as multiprocessing
+from multiprocessing import Process
+from superbench.benchmarks import BenchmarkRegistry
```

Comment:
Add blank line to separate native package and custom package. The common rule is:

import native package
blank line
import thirdparty pacakge
blank line
import custom package

Reply:
ok, thank you very much